assets/airports/eddm.json

```diff
@@ -1,883 +1,2695 @@
+                "26L": ["DM467", "DM466", "DM465", "DM464", "DM463", "DM452", "DM453", "DM454", ["DM455", "A80"], "DM456", "DM457", "DM458", "DM459"],
+                "26R": ["DM467", "DM466", "DM465", "DM464", "DM463", "DM452", "DM453", "DM454", ["DM455", "A80"], "DM456", "DM457", "DM458", "DM459"]
+            },
+            "draw": [
+                ["REDBU*", "NAPSA"],
+                ["AMADI*", "NAPSA"],
+                ["NAPSA", "DM458", "DM457", "DM456", "DM455", "DM454", "DM453", "DM452", "DM451", "DM450"],
+                ["NAPSA", "DM467", "DM466", "DM465", "DM464", "DM463", "DM452", "DM453", "DM454", "DM455", "DM456", "DM457", "DM458", "DM459"]
+            ]
+        },
+        "OBAGA4A": {
+            "icao": "OBAGA4A",
+            "name": "Obaga Four Alpha",
+            "entryPoints": {
+                "OBAGA": ["OBAGA"]
+            },
+            "body": ["NINUR", ["ANDEC", "A130|S280"], "BETOS"],
+            "rwy": {
+                "08L": ["DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", ["DM453", "A80"], "DM452", "DM451", "DM450"],
+                "08R": ["DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", ["DM453", "A80"], "DM452", "DM451", "DM450"],
+                "26L": ["DM452", "DM453", "DM454", ["DM455", "A80"], "DM456", "DM457", "DM458", "DM459"],
+                "26R": ["DM452", "DM453", "DM454", ["DM455", "A80"], "DM456", "DM457", "DM458", "DM459"]
+            },
+            "draw": [
+                ["OBAGA", "NINUR", "ANDEC", "BETOS"],
+                ["BETOS", "DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", "DM453", "DM452", "DM451", "DM450"],
+                ["BETOS", "DM452", "DM453", "DM454", "DM455", "DM456", "DM457", "DM458", "DM459"]
+            ]
+        },
+        "OSDER3A": {
+            "icao": "OSDER3A",
+            "name": "Osder Three Alpha",
+            "entryPoints": {
+                "OSDER": ["OSDER"]
+            },
+            "body": [["DISUN", "A120|S280"], "BETOS"],
+            "rwy": {
+                "08L": ["DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", ["DM453", "A80"], "DM452", "DM451", "DM450"],
+                "08R": ["DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", ["DM453", "A80"], "DM452", "DM451", "DM450"],
+                "26L": ["DM452", "DM453", "DM454", ["DM455", "A80"], "DM456", "DM457", "DM458", "DM459"],
+                "26R": ["DM452", "DM453", "DM454", ["DM455", "A80"], "DM456", "DM457", "DM458", "DM459"]
+            },
+            "draw": [
+                ["OSDER", "DISUN", "BETOS"],
+                ["BETOS", "DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", "DM453", "DM452", "DM451", "DM450"],
+                ["BETOS", "DM452", "DM453", "DM454", "DM455", "DM456", "DM457", "DM458", "DM459"]
+            ]
+        },
+        "ROKIL08": {
```

Comment:
All STARs will need a `rwy` section, populated with (at least) empty or populated arrays for each runway for which the procedure is valid.